datascience_eda/datascience_eda.py

```diff
@@ -165,3 +170,33 @@ def eda_text_columns(df, text_col = None, params = dict()):
     # plot a bar chart of most common tokens per entity
 
     # plot a bar chart of Part-of-speech tags
+
+
+def eda_categorical_columns(df, categorical_cols):
+    """Perfroms EDA of categorical features.
+    - Creates a dataframe containing column its details about unique values, null values and most frequenct category in the column
+    - Plots count plots for given categorical columns
+
+    Parameters
+    ----------
+    df : pandas.DataFrame
+        the dataset (X)
+    categorical_col : list
+        name of categorical column(s)
+        
+    Returns
+    -------
+    dataframe
+
+    Examples
+    -------
+    >>> eda_categorical_columns(X)
```

Comment:
there is no default value for categorical_cols to make it optional, so this code won't work.

Reply:
The idea was to make the user pass the columns he/she want.

Reply:
I understand that but you are not making the column list optional, so that example code will return an error.

Reply:
yes. i will change the example